### ISSUE TITLE
Animation.cs Easing Issue

### ISSUE THREAD

**Issue by erinachavez** [mattosaurus/ChartJSCore#38]
Trying to implement some custom animations for my charts, but I believe there is a bug in Animation.cs... Easing should be of "string" type, instead of "int?"?

```
/// <summary>
/// Easing function to use. Available options are: 'linear', 'easeInQuad', 'easeOutQuad', 'easeInOutQuad', 'easeInCubic', 'easeOutCubic', 'easeInOutCubic', 'easeInQuart', 'easeOutQuart', 'easeInOutQuart', 'easeInQuint', 'easeOutQuint', 'easeInOutQuint', 'easeInSine', 'easeOutSine', 'easeInOutSine', 'easeInExpo', 'easeOutExpo', 'easeInOutExpo', 'easeInCirc', 'easeOutCirc', 'easeInOutCirc', 'easeInElastic', 'easeOutElastic', 'easeInOutElastic', 'easeInBack', 'easeOutBack', 'easeInOutBack', 'easeInBounce', 'easeOutBounce', 'easeInOutBounce'.
/// </summary>
public int? Easing { get; set; }
```

Unless I'm misunderstanding, which I totally could be! Thank you!

**Comment by mattosaurus** [comment]
You're right, probably just a copy/paste mistake on my part.

Are you using a custom value (if possible) or one of the pre-defined ones?

https://www.chartjs.org/docs/latest/configuration/animations.html

Trying to figure out if this would be better done as an enum if it's only possible to use pre-defined values.

**Comment by erinachavez** [comment]
I was just trying to use a pre-defined one! The pre-defined values do seem to mirror css transitions, so I doubt you could create your own custom easing.

**Comment by mattosaurus** [comment]
I've created an enum to hold these values and have published an updated 1.5.2 package, hopefully that should do the job!

**Comment by erinachavez** [comment]
Works great! Thanks so much for the fast fix.
